src/Coordination/KeeperStorage.cpp

```diff
@@ -524,6 +526,7 @@ struct KeeperStorageRemoveRequestProcessor final : public KeeperStorageRequestPr
                 --parent.stat.numChildren;
                 ++parent.stat.cversion;
                 parent.children.erase(child_basename);
+                parent.size_bytes -= child_basename.size();
```

Comment:
But what about `set` request when `size` of `data` changes?

Reply:
done

Reply:
There is no need to update size in set acl processor because of acl is fixed width.